package/dataset-librispeech/install.bat

```diff
@@ -0,0 +1,12 @@
+
+echo Downloading COCO 2017 validation dataset
```

Comment:
Remove `install.bat` altogether as not supported?

Reply:
Done